### PR TITLE
remove azure dependency, replaced by minio and docker secret

### PR DESCRIPTION
**Pull request by Nathan-Kr**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,11 +4,10 @@ COPY package.json /app
...
-RUN yarn install
-RUN yarn build
+RUN npm install
+RUN npm run build
 
 EXPOSE 3000
 
-CMD ["yarn", "start"]
\ No newline at end of file
+CMD ["npm", "start"]
```
Comment (follyjohn) [inline Dockerfile]: Yarn not npm

File: src/config/environnement.ts [hunk 2]
```diff
@@ -14,23 +14,12 @@ export const environnement = {
...
-	jwt_secret_name: process.env.JWT_SECRET_KEY_NAME || 'jwtsecret',
+	jwt_secret: process.env.JWT_SECRET || 'jwtsecret',
+	jwt_secret_file: process.env.JWT_SECRET_FILE || '/run/secrets/jwt_secret',
 	jwt_token_expiration_time: process.env.JWT_TOKEN_EXPIRATION_TIME || '2 days',
 	jwt_token_issuer: process.env.JWT_ISSUER_NAME || 'neighbook-api',
 	jwt_token_audience: process.env.JWT_AUDIENCE_NAME || 'neighbook-api',
 	argon2_salt_key_name: process.env.ARGON2_SECRET_KEY_NAME || 'argon2secret',
-	storage_accepted_files_extensions: process.env.STORAGE_ACCEPTED_FILES_EXTENSIONS || 'jpg,jpeg,png,gif',
-	storage_accepted_files_max_size: process.env.STORAGE_ACCEPTED_FILES_MAX_SIZE || 1000000,
-	storage_accepted_files_types: process.env.STORAGE_ACCEPTED_FILES_TYPES || 'image/jpeg,image/png,image/gif',
```
Comment (follyjohn) [inline src/config/environnement.ts:24]: We have to keep storage setup like files max size

File: src/doc/swagger.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import swaggerAutogen from 'swagger-autogen';
 
 import { metadata } from '../config/openapi';
 
-const outputFile = './openapi.json';
+const outputFile = './dist/doc/openapi.json';
```
Comment (follyjohn) [inline src/doc/swagger.ts]: We have a command to do that => refer to README.md

**Review by follyjohn — CHANGES_REQUESTED** [review]

File: src/doc/swagger.ts (hunk 3 above)
Comment (Nathan-Kr) [reply src/doc/swagger.ts]: si je met pas ca, ca genere la doc a la racine du projet

File: src/config/environnement.ts (hunk 2 above)
Comment (Nathan-Kr) [reply src/config/environnement.ts:24]: deplacé ligne 43 section storage

File: src/doc/swagger.ts (hunk 3 above)
Comment (follyjohn) [reply src/doc/swagger.ts]: C'est le but en Dev, Dist c'est pour la prod (build)

**Review by follyjohn — APPROVED** [review]
LGTM

**Pull request merged**